source/read/change-streams.txt

```diff
@@ -0,0 +1,261 @@
+.. _php-change-streams:
+
+====================
+Monitor Data Changes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: watch, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use a **change stream** to monitor real-time
+changes to your data. A change stream is a {+mdb-server+} feature that
+allows your application to subscribe to data changes on a collection, database,
+or deployment.
+
+When using the {+php-library+}, you can create an instance of ``MongoDB\ChangeStream`` to
+monitor data changes, such as updates, insertions, and deletions.
```

Comment:
Users never construct a `MongoDB\ChangeStream` object. It'd be more correct to say they can invoke the various `watch()` methods on Client, Database, or Collection objects to obtain a ChangeStream object that they can then iterate to monitor data changes on the logical entity (cluster, database, or collection, respectively).